pip/requirements.txt

```diff
@@ -298,7 +298,7 @@ tqdm==4.62.2
 traitlets==5.1.0
 typed-ast==1.4.3
 typing-extensions==3.10.0.2
-typing==3.7.4.3
```

Comment:
Did we not drop `py3-typing` some time ago? If not, we should:

> NOTE: in Python 3.5 and later, the typing module lives in the stdlib, and installing this package has NO EFFECT

(from https://pypi.org/project/typing/)

Reply:
I do not remember such a change. Yes, we should drop it as it is picked up from python3 installations
```
[muzaffar@lxplus796 ~]$ source /cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/py3-typing/3.7.4.3/etc/profile.d/init.sh
[muzaffar@lxplus796 ~]$ python3 -c 'import typing;print(typing.__file__)'
/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/python3/3.8.2-llifpc/lib/python3.8/typing.py
```

@aandvalenzuela , can you please remove it from requirements.txt and also remove all of its usages e.g. https://github.com/cms-sw/cmsdist/search?q=py3-typing